test/modules/change-email/controller.test.js

```diff
@@ -14,7 +14,8 @@ const idm = require('../../../src/lib/connectors/idm')
 const crmEntities = require('../../../src/lib/connectors/crm/entities')
 const notifications = require('../../../src/lib/notifications/emails')
 const { logger } = require('../../../src/logger')
-const event = require('../../../src/lib/event')
+// const event = require('../../../src/lib/event')
```

Comment:
did you mean to leave this commented out or included?

Reply:
Nope! Good spot!